app/assets/javascripts/app/validate-field.js

```diff
@@ -27,6 +27,7 @@ function addInvalidMarkup(f) {
 
 function removeInvalidMarkup(f) {
   f.parentNode.classList.remove('has-error');
+  f.parentNode.classList.remove('invalid');
```

Comment:
I'm not sure this is necessary - the h5f library automatically removes this class from the field when it's valid and the focus is removed from the field.